vmdk_plugin/Makefile

```diff
@@ -350,7 +350,7 @@ test-all:  test e2e-test testremote
 # An end-to-end test target for vSphere Docker Volume Driver
 e2e-test:
 	$(log_target)
-	$(SCRIPTS)/build.sh e2e-dkrVolDriver-test "$(ESX)" "$(VM1)" "$(VM2)"
+	$(SCRIPTS)/build.sh e2e-dkrVolDriver-test "$(ESX)" "$(VM1)" "$(VM2)" "$(DS1)" "$(DS2)" "$(CNAME)" "$(T1)"
```

Comment:
sharing some command for your reference. 

`govc vm.info -json <VM_NAME> | jq -r .VirtualMachines[].Config.DatastoreUrl[].Name`: grabbing VM's datastore value.

`govc datastore.info`: returns all available datastores in the inventory

sample output: VM on my local testbed is "photon1" created on "TestDatastore"
```
nshukla-mbp:~ nshukla$ govc vm.info -json photon1 | jq -r .VirtualMachines[].Config.DatastoreUrl[].Name
TestDatastore
nshukla-mbp:~ nshukla$ govc datastore.info
Name:        TestDatastore
  Path:      /ha-datacenter/datastore/TestDatastore
  Type:      VMFS
  URL:       /vmfs/volumes/4fc608aa-828f03c0-d4a2-d4ae52794a7a
  Capacity:  460.8 GB
  Free:      415.1 GB
```

We have just started adding some handy utils at https://github.com/vmware/docker-volume-vsphere/blob/master/tests/utils/govc/cmd.go

Let me know if there are any issues. I can supply a quick util if you want as a separate PR. I will be looking forward to your response.